docs/roomHandlers/fishingRoom.md

```diff
@@ -0,0 +1,41 @@
+# Fishing Room
+
+A room with a beach and a boat dock. The player can fish from dock or leave. Fishing may result in catching a fish.
+
+## States
+
+- **AtEntrance**: The player is on a dock with a fishing pole
+- **Fishing**: The player casts their fishing line in hopes to find a fish
+- **Caught**: The player now has a fish
+- **BrokenLine**: The player has a broken their fishing line and the fish got away
+- **FixLine**: The player can fix their broken line to fish again
+
+## Transitions
+
+- **leave**: The player leaves through the only available door
```

Comment:
What do you think about having the `leave` command output how many fish you caught, since we don't have an inventory yet.

Reply:
updated